app/controllers/avo/actions_controller.rb

```diff
@@ -17,6 +17,7 @@ def show
       @view = Avo::ViewInquirer.new("new")
 
       @resource.hydrate(record: @record, view: @view, user: _current_user, params: params)
+      @fields = @action.get_fields.reject { |field| field.is_a?(Avo::Fields::HiddenField) }
```

Comment:
We want to render all fields on show, even the hidden ones.

```suggestion
      @fields = @action.get_fields
```

Reply:
Yes. I will change it.